pages/linux/ip.md

```diff
@@ -6,6 +6,14 @@
 
 `ip a`
 
+- List interfaces with brief network layer info:
+
+`ip -brief a`
```

Comment:
Perhaps we could use the full name instead of just `a` here? We generally prefer that in tldr pages as it aids clarity.

I know that the rest of the page follows the trend of shortening everything, but it would be awesome if we could change that while we're at it :-)
```suggestion
`ip -brief address`
```

Reply:
I actually _do_ agree with you on this, I just tried to follow the trend. I'll apply it soon.

Reply:
I have pushed it. Also added a refactor patch of the other examples in the file.